app/services/idv/actions/redo_address_action.rb

```diff
@@ -6,6 +6,7 @@ def self.analytics_submitted_event
       end
 
       def call
+        mark_step_incomplete(:address)
```

Comment:
Non-blocking request: Would it be possible to add a spec for this action?  Again, this is non-blocking.

Reply:
I think it's possible. I'm going to wait until I hear back from team Ada.

Reply:
No need as this change is uneccessary.